string current_region
---
string next_region
